masterlist.yaml

```diff
@@ -804,6 +822,7 @@ plugins:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'RealisticWaterTwo.esp'
+    #global_priority: 72
```

Comment:
Looks like you left this in by accident.

Reply:
yeah, will remove that.